lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-permalink.tpl.php

```diff
@@ -40,10 +40,10 @@
               <?php print filter_xss($rb['caption']); ?> - <?php print check_plain($user->first_name); ?>
             </div>
           <?php endif; ?>
-          <?php if (isset($kudos) && !$kudos['disable_reactions']): ?>
-            <ul class="form-actions -inline kudos" <?php print $kudos['rbid'] ? 'data-reportback-item-id="' . $kudos['rbid'] . '"' : ''; ?>>
+          <?php if (isset($content['kudos']) && isset($content['kudos']['fid'])): ?>
+            <ul class="form-actions -inline kudos" <?php print $kudos['fid'] ? 'data-reportback-item-id="' . $kudos['fid'] . '"' : ''; ?>>
```

Comment:
since this is now in the `$content` variable... how does `$kudos['fid']` know that it's referring to `$content['kudos']['fid']`?